--- codeblocks-17.12/m4/acinclude.m4.orig	2019-04-19 18:32:27.583835515 +0200
+++ codeblocks-17.12/m4/acinclude.m4	2019-04-19 18:33:27.033513448 +0200
@@ -97,8 +97,8 @@
         AC_MSG_RESULT(yes)
     else
         AS_IF([test "x$enable_flags_setting" = "xyes"],[
-            CFLAGS="-O2 -ffast-math $CFLAGS"
-            CXXFLAGS="-O2 -ffast-math $CXXFLAGS"
+            CFLAGS="-Wall -ffast-math $CFLAGS"
+            CXXFLAGS="-Wall -ffast-math $CXXFLAGS"
         ], [
             : ${CFLAGS=""}
             : ${CXXFLAGS=""}
