diff -urN codeblocks-8.02_orig/src/sdk/configmanager.cpp codeblocks-8.02/src/sdk/configmanager.cpp
--- codeblocks-8.02_orig/src/sdk/configmanager.cpp.orig	2010-05-22 12:10:05.000000000 +0200
+++ codeblocks-8.02_orig/src/sdk/configmanager.cpp	2010-07-01 01:17:20.000000000 +0200
@@ -514,7 +514,8 @@ wxString ConfigManager::GetFolder(Search
 
         case sdPluginsGlobal:
 #ifndef CB_AUTOCONF
-            return ConfigManager::data_path_global + wxFILE_SEP_PATH + _T("plugins");
+            // return ConfigManager::data_path_global + wxFILE_SEP_PATH + _T("plugins");
+            return _T("@libdir@/codeblocks/plugins");
 #else
             return ConfigManager::plugin_path_global;
 #endif
