diff -urN codeblocks-8.02/acinclude.m4 codeblocks-8.02.new/acinclude.m4
--- codeblocks-8.02/acinclude.m4	2009-01-30 20:16:37.000000000 +0100
+++ codeblocks-8.02.new/acinclude.m4	2009-01-30 22:59:03.000000000 +0100
@@ -71,8 +71,8 @@
          CXXFLAGS="-g -DDEBUG $CXXFLAGS"
 	AC_MSG_RESULT(yes)
 else
-	CFLAGS="-O2 -ffast-math $CFLAGS"
-	CXXFLAGS="-O2 -ffast-math $CXXFLAGS"
+	CFLAGS="-Wall -ffast-math $CFLAGS"
+	CXXFLAGS="-Wall -ffast-math $CXXFLAGS"
 	AC_MSG_RESULT(no)
 fi
 
