diff -ur codeblocks-13.12/src/plugins/abbreviations/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/abbreviations/resources/Makefile.am
--- codeblocks-13.12/src/plugins/abbreviations/resources/Makefile.am	2013-11-17 20:33:51.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/abbreviations/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -4,5 +4,5 @@
 EXTRA_DIST = manifest.xml $(srcdir)/*.xrc
 
 abbreviations.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/abbreviations.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/abbreviations.zip manifest.xml *.xrc > /dev/null
 
diff -ur codeblocks-13.12/src/plugins/astyle/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/astyle/resources/Makefile.am
--- codeblocks-13.12/src/plugins/astyle/resources/Makefile.am	2013-11-17 20:34:12.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/astyle/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 astyle.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/astyle.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/astyle.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/autosave/Makefile.am codeblocks-13.12.pwd/src/plugins/autosave/Makefile.am
--- codeblocks-13.12/src/plugins/autosave/Makefile.am	2013-11-17 20:33:50.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/autosave/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -17,4 +17,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 autosave.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/autosave.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/autosave.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/classwizard/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/classwizard/resources/Makefile.am
--- codeblocks-13.12/src/plugins/classwizard/resources/Makefile.am	2013-11-17 20:33:50.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/classwizard/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 classwizard.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/classwizard.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/classwizard.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/codecompletion/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/codecompletion/resources/Makefile.am
--- codeblocks-13.12/src/plugins/codecompletion/resources/Makefile.am	2013-11-17 20:34:13.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/codecompletion/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -56,7 +56,7 @@
              images/vars_folder.png
 
 codecompletion.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/codecompletion.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/codecompletion.zip manifest.xml *.xrc > /dev/null
 
 install-exec-local:
 	mkdir -p $(DESTDIR)$(pkgdatadir)/images/codecompletion
diff -ur codeblocks-13.12/src/plugins/compilergcc/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/compilergcc/resources/Makefile.am
--- codeblocks-13.12/src/plugins/compilergcc/resources/Makefile.am	2013-11-17 20:33:51.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/compilergcc/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -20,7 +20,7 @@
 	images/16x16/stop.png
 
 compiler.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/compiler.zip manifest.xml *.xrc images/*.png images/16x16/*.png > /dev/null
+	zip $(PWD)/compiler.zip manifest.xml *.xrc images/*.png images/16x16/*.png > /dev/null
 
 compilers_xmldir = $(datadir)/@PACKAGE@/compilers
 
diff -ur codeblocks-13.12/src/plugins/contrib/AutoVersioning/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/AutoVersioning/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/AutoVersioning/Makefile.am	2013-11-17 20:34:10.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/AutoVersioning/Makefile.am	2014-01-18 19:35:30.555377867 +0100
@@ -33,4 +33,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 AutoVersioning.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/AutoVersioning.zip manifest.xml > /dev/null
+	zip $(PWD)/AutoVersioning.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/BrowseTracker/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/BrowseTracker/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/BrowseTracker/Makefile.am	2013-11-17 20:34:10.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/BrowseTracker/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -42,4 +42,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 BrowseTracker.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/BrowseTracker.zip manifest.xml > /dev/null
+	zip $(PWD)/BrowseTracker.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/byogames/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/byogames/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/byogames/Makefile.am	2013-11-17 20:34:09.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/byogames/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -35,4 +35,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 byogames.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/byogames.zip manifest.xml > /dev/null
+	zip $(PWD)/byogames.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/cb_koders/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/cb_koders/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/cb_koders/Makefile.am	2013-11-17 20:33:55.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/cb_koders/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -24,4 +24,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 cb_koders.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/cb_koders.zip manifest.xml > /dev/null
+	zip $(PWD)/cb_koders.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/Cccc/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/Cccc/resources/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/Cccc/resources/Makefile.am	2013-11-17 20:34:04.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/Cccc/resources/Makefile.am	2014-01-18 19:35:30.555377867 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 Cccc.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/Cccc.zip manifest.xml > /dev/null
+	zip $(PWD)/Cccc.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/codesnippets/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/codesnippets/resources/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/codesnippets/resources/Makefile.am	2013-11-17 20:34:08.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/codesnippets/resources/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -140,5 +140,5 @@
 EXTRA_DIST = manifest.xml
 
 codesnippets.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/codesnippets.zip manifest.xml > /dev/null
+	zip $(PWD)/codesnippets.zip manifest.xml > /dev/null
 
diff -ur codeblocks-13.12/src/plugins/contrib/codestat/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/codestat/resources/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/codestat/resources/Makefile.am	2013-11-17 20:34:09.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/codestat/resources/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 codestat.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/codestat.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/codestat.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/copystrings/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/copystrings/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/copystrings/Makefile.am	2013-11-17 20:33:55.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/copystrings/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -22,4 +22,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 copystrings.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/copystrings.zip manifest.xml > /dev/null
+	zip $(PWD)/copystrings.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/CppCheck/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/CppCheck/resources/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/CppCheck/resources/Makefile.am	2013-11-17 20:33:54.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/CppCheck/resources/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 CppCheck.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/CppCheck.zip manifest.xml > /dev/null
+	zip $(PWD)/CppCheck.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/Cscope/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/Cscope/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/Cscope/Makefile.am	2013-11-17 20:34:00.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/Cscope/Makefile.am	2014-01-18 19:35:30.555377867 +0100
@@ -36,4 +36,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 Cscope.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/Cscope.zip manifest.xml > /dev/null
+	zip $(PWD)/Cscope.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/DoxyBlocks/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/DoxyBlocks/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/DoxyBlocks/Makefile.am	2013-11-17 20:33:54.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/DoxyBlocks/Makefile.am	2014-01-18 19:35:30.555377867 +0100
@@ -41,7 +41,7 @@
 CLEANFILES = $(pkgdata_DATA)
 
 DoxyBlocks.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip -R $(PWD)/DoxyBlocks.zip manifest.xml images/*.png images/16x16/*.png > /dev/null
+	zip -R $(PWD)/DoxyBlocks.zip manifest.xml images/*.png images/16x16/*.png > /dev/null
 
 confimagesdir = $(datadir)/@PACKAGE@/images/settings
 
diff -ur codeblocks-13.12/src/plugins/contrib/dragscroll/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/dragscroll/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/dragscroll/Makefile.am	2013-11-17 20:34:05.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/dragscroll/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -26,4 +26,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 dragscroll.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/dragscroll.zip manifest.xml > /dev/null
+	zip $(PWD)/dragscroll.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/EditorConfig/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/EditorConfig/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/EditorConfig/Makefile.am	2013-11-17 20:34:00.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/EditorConfig/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -26,4 +26,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 EditorConfig.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/EditorConfig.zip manifest.xml > /dev/null
+	zip $(PWD)/EditorConfig.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/EditorTweaks/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/EditorTweaks/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/EditorTweaks/Makefile.am	2013-11-17 20:34:09.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/EditorTweaks/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -26,4 +26,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 EditorTweaks.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/EditorTweaks.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/EditorTweaks.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/envvars/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/envvars/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/envvars/Makefile.am	2013-11-17 20:33:54.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/envvars/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -32,4 +32,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 envvars.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/envvars.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/envvars.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/FileManager/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/FileManager/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/FileManager/Makefile.am	2013-11-17 20:33:59.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/FileManager/Makefile.am	2014-01-18 19:35:30.555377867 +0100
@@ -41,4 +41,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 FileManager.zip: manifest.xml Resources/fileexplorersettings.xrc
-	PWD=`pwd` cd $(srcdir) && zip -j9 $(PWD)/FileManager.zip manifest.xml Resources/fileexplorersettings.xrc > /dev/null
+	zip -j9 $(PWD)/FileManager.zip manifest.xml Resources/fileexplorersettings.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/headerfixup/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/headerfixup/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/headerfixup/Makefile.am	2013-11-17 20:34:08.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/headerfixup/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -40,4 +40,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 headerfixup.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/headerfixup.zip manifest.xml > /dev/null
+	zip $(PWD)/headerfixup.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/help_plugin/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/help_plugin/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/help_plugin/Makefile.am	2013-11-17 20:34:09.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/help_plugin/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -61,4 +61,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 help_plugin.zip: manifest.xml help_plugin.xrc
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/help_plugin.zip manifest.xml *.xrc *.png > /dev/null
+	zip $(PWD)/help_plugin.zip manifest.xml *.xrc *.png > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/HexEditor/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/HexEditor/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/HexEditor/Makefile.am	2013-11-17 20:33:55.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/HexEditor/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -66,4 +66,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 HexEditor.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/HexEditor.zip manifest.xml > /dev/null
+	zip $(PWD)/HexEditor.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/IncrementalSearch/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/IncrementalSearch/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/IncrementalSearch/Makefile.am	2013-11-17 20:34:02.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/IncrementalSearch/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -29,7 +29,7 @@
 CLEANFILES = $(pkgdata_DATA)
 
 IncrementalSearch.zip: manifest.xml IncrementalSearchToolbar.xrc IncrementalSearchConfDlg.xrc
-	PWD=`pwd` cd $(srcdir) && zip -R $(PWD)/IncrementalSearch.zip manifest.xml *.xrc images/*.png images/16x16/*.png > /dev/null
+	zip -R $(PWD)/IncrementalSearch.zip manifest.xml *.xrc images/*.png images/16x16/*.png > /dev/null
 
 confimagesdir = $(datadir)/@PACKAGE@/images/settings
 
diff -ur codeblocks-13.12/src/plugins/contrib/keybinder/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/keybinder/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/keybinder/Makefile.am	2013-11-17 20:34:10.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/keybinder/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -26,4 +26,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 keybinder.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/keybinder.zip manifest.xml > /dev/null
+	zip $(PWD)/keybinder.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/lib_finder/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/lib_finder/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/lib_finder/Makefile.am	2013-11-17 20:34:06.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/lib_finder/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -71,4 +71,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 lib_finder.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/lib_finder.zip manifest.xml > /dev/null
+	zip $(PWD)/lib_finder.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/MouseSap/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/MouseSap/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/MouseSap/Makefile.am	2013-11-17 20:34:04.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/MouseSap/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -23,4 +23,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 MouseSap.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/MouseSap.zip manifest.xml > /dev/null
+	zip $(PWD)/MouseSap.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/NassiShneiderman/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/NassiShneiderman/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/NassiShneiderman/Makefile.am	2013-11-17 20:34:04.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/NassiShneiderman/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -81,4 +81,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 NassiShneiderman.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/NassiShneiderman.zip manifest.xml > /dev/null
+	zip $(PWD)/NassiShneiderman.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/profiler/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/profiler/resources/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/profiler/resources/Makefile.am	2013-11-17 20:33:54.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/profiler/resources/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 Profiler.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/Profiler.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/Profiler.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/regex_testbed/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/regex_testbed/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/regex_testbed/Makefile.am	2013-11-17 20:33:55.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/regex_testbed/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -23,4 +23,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 RegExTestbed.zip: manifest.xml regexdlg.xrc
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/RegExTestbed.zip manifest.xml regexdlg.xrc > /dev/null
+	zip $(PWD)/RegExTestbed.zip manifest.xml regexdlg.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/ReopenEditor/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/ReopenEditor/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/ReopenEditor/Makefile.am	2013-11-17 20:34:06.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/ReopenEditor/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -28,4 +28,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 ReopenEditor.zip: manifest.xml ReopenEditorConfDLg.xrc
-	PWD=`pwd` cd $(srcdir) && zip -R $(PWD)/ReopenEditor.zip manifest.xml  *.xrc > /dev/null
+	zip -R $(PWD)/ReopenEditor.zip manifest.xml  *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/SmartIndent/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/SmartIndent/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/SmartIndent/Makefile.am	2013-11-17 20:34:04.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/SmartIndent/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -102,37 +102,30 @@
 .NOTPARALLEL: $(pkgdata_DATA)
 
 SmartIndentHDL.zip: manifest_hdl.xml
-	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_hdl.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentHDL.zip manifest.xml
 
 SmartIndentPascal.zip: manifest_pascal.xml
-	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_pascal.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentPascal.zip manifest.xml
 
 SmartIndentCpp.zip: manifest_cpp.xml
-	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_cpp.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentCpp.zip manifest.xml
 
 SmartIndentPython.zip: manifest_python.xml
-	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_python.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentPython.zip manifest.xml
 
 SmartIndentLua.zip: manifest_lua.xml
-	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_lua.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentLua.zip manifest.xml
 
 SmartIndentFortran.zip: manifest_fortran.xml
-	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_fortran.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentFortran.zip manifest.xml
 
 SmartIndentXML.zip: manifest_xml.xml
-	PWD=`pwd` cd $(srcdir) && \
 	cp manifest_xml.xml ./manifest.xml && \
 	zip -R $(PWD)/SmartIndentXML.zip manifest.xml
 
diff -ur codeblocks-13.12/src/plugins/contrib/source_exporter/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/source_exporter/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/source_exporter/Makefile.am	2013-11-17 20:34:03.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/source_exporter/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -38,4 +38,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 exporter.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/exporter.zip manifest.xml > /dev/null
+	zip $(PWD)/exporter.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/SpellChecker/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/SpellChecker/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/SpellChecker/Makefile.am	2013-11-17 20:34:09.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/SpellChecker/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -57,7 +57,7 @@
 CLEANFILES = $(pkgdata_DATA)
 
 SpellChecker.zip: manifest.xml SpellCheckSettingsPanel.xrc
-	PWD=`pwd` cd $(srcdir) && zip -R $(PWD)/SpellChecker.zip manifest.xml SpellCheckSettingsPanel.xrc > /dev/null
+	zip -R $(PWD)/SpellChecker.zip manifest.xml SpellCheckSettingsPanel.xrc > /dev/null
 
 confimagesdir = $(datadir)/@PACKAGE@/images/settings
 
diff -ur codeblocks-13.12/src/plugins/contrib/symtab/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/symtab/resources/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/symtab/resources/Makefile.am	2013-11-17 20:34:06.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/symtab/resources/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 SymTab.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/SymTab.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/SymTab.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/ThreadSearch/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/ThreadSearch/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/ThreadSearch/Makefile.am	2013-11-17 20:34:05.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/ThreadSearch/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -71,5 +71,5 @@
 CLEANFILES = $(pkgdata_DATA)
 
 ThreadSearch.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/ThreadSearch.zip manifest.xml > /dev/null
+	zip $(PWD)/ThreadSearch.zip manifest.xml > /dev/null
 
diff -ur codeblocks-13.12/src/plugins/contrib/ToolsPlus/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/ToolsPlus/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/ToolsPlus/Makefile.am	2013-11-17 20:34:08.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/ToolsPlus/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -42,5 +42,5 @@
 CLEANFILES = $(pkgdata_DATA)
 
 ToolsPlus.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip -j9 $(PWD)/ToolsPlus.zip manifest.xml > /dev/null
+	zip -j9 $(PWD)/ToolsPlus.zip manifest.xml > /dev/null
 
diff -ur codeblocks-13.12/src/plugins/contrib/Valgrind/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/Valgrind/resources/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/Valgrind/resources/Makefile.am	2013-11-17 20:34:10.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/Valgrind/resources/Makefile.am	2014-01-18 19:35:30.552044534 +0100
@@ -5,7 +5,7 @@
 CLEANFILES = $(pkgdata_DATA)
 
 Valgrind.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/Valgrind.zip manifest.xml > /dev/null
+	zip $(PWD)/Valgrind.zip manifest.xml > /dev/null
 
 confimagesdir = $(datadir)/@PACKAGE@/images/settings
 
diff -ur codeblocks-13.12/src/plugins/contrib/wxSmith/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/wxSmith/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/wxSmith/Makefile.am	2013-11-17 20:33:59.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/wxSmith/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -85,4 +85,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 wxsmith.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/wxsmith.zip manifest.xml > /dev/null
+	zip $(PWD)/wxsmith.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/plugins/contrib/wxSmithAui/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/wxSmithAui/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/wxSmithAui/Makefile.am	2013-11-17 20:33:55.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/wxSmithAui/Makefile.am	2014-01-18 19:35:30.548711201 +0100
@@ -28,7 +28,7 @@
 CLEANFILES = $(pkgdata_DATA)
 
 wxSmithAui.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/wxSmithAui.zip manifest.xml > /dev/null
+	zip $(PWD)/wxSmithAui.zip manifest.xml > /dev/null
 
 libwxSmithAui_la_SOURCES = \
 ./wxAuiManager/wxsAuiManager.cpp \
diff -ur codeblocks-13.12/src/plugins/contrib/wxSmithContribItems/Makefile.am codeblocks-13.12.pwd/src/plugins/contrib/wxSmithContribItems/Makefile.am
--- codeblocks-13.12/src/plugins/contrib/wxSmithContribItems/Makefile.am	2013-11-17 20:34:01.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/contrib/wxSmithContribItems/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -57,7 +57,7 @@
 CLEANFILES = $(pkgdata_DATA)
 
 wxsmithcontribitems.zip: manifest.xml
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/wxsmithcontribitems.zip manifest.xml > /dev/null
+	zip $(PWD)/wxsmithcontribitems.zip manifest.xml > /dev/null
 
 libwxsmithcontribitems_la_SOURCES = \
 ./wxflatnotebook/wxsFlatNotebook.cpp \
diff -ur codeblocks-13.12/src/plugins/debuggergdb/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/debuggergdb/resources/Makefile.am
--- codeblocks-13.12/src/plugins/debuggergdb/resources/Makefile.am	2013-11-17 20:34:12.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/debuggergdb/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -10,4 +10,4 @@
 EXTRA_DIST = $(DEBUG_RESOURCES)
 
 debugger.zip: $(DEBUG_RESOURCES)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/debugger.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/debugger.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/defaultmimehandler/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/defaultmimehandler/resources/Makefile.am
--- codeblocks-13.12/src/plugins/defaultmimehandler/resources/Makefile.am	2013-11-17 20:33:51.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/defaultmimehandler/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -4,5 +4,5 @@
 EXTRA_DIST = edit_files_handling.xrc manifest.xml
 
 defaultmimehandler.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/defaultmimehandler.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/defaultmimehandler.zip manifest.xml *.xrc > /dev/null
 
diff -ur codeblocks-13.12/src/plugins/occurrenceshighlighting/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/occurrenceshighlighting/resources/Makefile.am
--- codeblocks-13.12/src/plugins/occurrenceshighlighting/resources/Makefile.am	2013-11-17 20:34:12.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/occurrenceshighlighting/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -4,4 +4,4 @@
 EXTRA_DIST = manifest.xml $(srcdir)/*.xrc
 
 occurrenceshighlighting.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/occurrenceshighlighting.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/occurrenceshighlighting.zip manifest.xml *.xrc > /dev/null
diff -ur codeblocks-13.12/src/plugins/openfileslist/Makefile.am codeblocks-13.12.pwd/src/plugins/openfileslist/Makefile.am
--- codeblocks-13.12/src/plugins/openfileslist/Makefile.am	2013-11-17 20:33:50.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/openfileslist/Makefile.am	2014-01-18 19:35:30.542044534 +0100
@@ -19,5 +19,5 @@
 EXTRA_DIST = manifest.xml
 
 openfileslist.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/openfileslist.zip manifest.xml > /dev/null
+	zip $(PWD)/openfileslist.zip manifest.xml > /dev/null
 
diff -ur codeblocks-13.12/src/plugins/projectsimporter/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/projectsimporter/resources/Makefile.am
--- codeblocks-13.12/src/plugins/projectsimporter/resources/Makefile.am	2013-11-17 20:34:14.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/projectsimporter/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -4,5 +4,5 @@
 EXTRA_DIST = project_import_menu.xrc manifest.xml
 
 projectsimporter.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/projectsimporter.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/projectsimporter.zip manifest.xml *.xrc > /dev/null
 
diff -ur codeblocks-13.12/src/plugins/scriptedwizard/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/scriptedwizard/resources/Makefile.am
--- codeblocks-13.12/src/plugins/scriptedwizard/resources/Makefile.am	2013-11-17 20:33:54.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/scriptedwizard/resources/Makefile.am	2014-01-18 19:35:30.542044534 +0100
@@ -11,5 +11,5 @@
 EXTRA_DIST = manifest.xml
 
 scriptedwizard.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/scriptedwizard.zip manifest.xml > /dev/null
+	zip $(PWD)/scriptedwizard.zip manifest.xml > /dev/null
 
diff -ur codeblocks-13.12/src/plugins/todo/resources/Makefile.am codeblocks-13.12.pwd/src/plugins/todo/resources/Makefile.am
--- codeblocks-13.12/src/plugins/todo/resources/Makefile.am	2013-11-17 20:34:13.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/todo/resources/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -4,5 +4,5 @@
 EXTRA_DIST = manifest.xml $(srcdir)/*.xrc
 
 todo.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/todo.zip manifest.xml *.xrc > /dev/null
+	zip $(PWD)/todo.zip manifest.xml *.xrc > /dev/null
 
diff -ur codeblocks-13.12/src/plugins/xpmanifest/Makefile.am codeblocks-13.12.pwd/src/plugins/xpmanifest/Makefile.am
--- codeblocks-13.12/src/plugins/xpmanifest/Makefile.am	2013-11-17 20:34:14.000000000 +0100
+++ codeblocks-13.12.pwd/src/plugins/xpmanifest/Makefile.am	2014-01-18 19:35:30.545377868 +0100
@@ -18,4 +18,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 xpmanifest.zip: $(EXTRA_DIST)
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/xpmanifest.zip manifest.xml > /dev/null
+	zip $(PWD)/xpmanifest.zip manifest.xml > /dev/null
diff -ur codeblocks-13.12/src/sdk/resources/Makefile.am codeblocks-13.12.pwd/src/sdk/resources/Makefile.am
--- codeblocks-13.12/src/sdk/resources/Makefile.am	2013-11-17 20:33:47.000000000 +0100
+++ codeblocks-13.12.pwd/src/sdk/resources/Makefile.am	2014-01-18 19:35:30.555377867 +0100
@@ -9,4 +9,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 manager_resources.zip: $(dist_noinst_DATA)
-	PWD=`pwd` cd $(srcdir) && zip -qu $(PWD)/manager_resources.zip *.xrc images/*.png images/16x16/*.png images/12x12/*.png > /dev/null
+	zip -qu $(PWD)/manager_resources.zip *.xrc images/*.png images/16x16/*.png images/12x12/*.png > /dev/null
diff -ur codeblocks-13.12/src/src/resources/Makefile.am codeblocks-13.12.pwd/src/src/resources/Makefile.am
--- codeblocks-13.12/src/src/resources/Makefile.am	2013-11-17 20:33:49.000000000 +0100
+++ codeblocks-13.12.pwd/src/src/resources/Makefile.am	2014-01-18 19:35:30.538711201 +0100
@@ -9,7 +9,7 @@
 CLEANFILES = $(dist_pkgdata_DATA)
 
 resources.zip: $(srcdir)/*.xrc $(srcdir)/images/*.png $(srcdir)/images/16x16/*.png
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/resources.zip *.xrc images/*.png images/16x16/*.png > /dev/null
+	zip $(PWD)/resources.zip *.xrc images/*.png images/16x16/*.png > /dev/null
 
 start_here.zip: $(srcdir)/start_here/*.htm* $(srcdir)/start_here/*.png
 	PWD=`pwd` cd $(srcdir)/start_here && zip $(PWD)/start_here.zip *.htm* *.png > /dev/null
