--- codeblocks-1.0rc2/src/sdk/resources/Makefile.am.orig	2006-01-16 16:57:32.000000000 +0100
+++ codeblocks-1.0rc2/src/sdk/resources/Makefile.am	2006-01-16 16:59:02.000000000 +0100
@@ -7,4 +7,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 manager_resources.zip:
-	PWD=`pwd` cd $(srcdir) && zip -j $(PWD)/manager_resources.zip *.xrc images/*.png > /dev/null
+	zip -j manager_resources.zip *.xrc images/*.png > /dev/null
--- codeblocks-1.0rc2/src/src/resources/Makefile.am.orig	2006-03-06 17:37:21.000000000 +0100
+++ codeblocks-1.0rc2/src/src/resources/Makefile.am	2006-03-06 17:39:05.000000000 +0100
@@ -9,7 +9,9 @@
 CLEANFILES = $(pkgdata_DATA)
 
 resources.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/resources.zip *.xrc images/*.png images/16x16/*.png > /dev/null
+	zip resources.zip *.xrc images/*.png images/16x16/*.png > /dev/null
 
 start_here.zip:
-	PWD=`pwd` cd $(srcdir)/start_here && zip $(PWD)/start_here.zip *.htm* *.png > /dev/null
+	cd start_here && \
+	zip ../start_here.zip *.htm* *.png > /dev/null && \
+	cd ..
--- codeblocks-1.0rc2/src/plugins/todo/resources/Makefile.am.orig	2006-03-06 17:37:08.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/todo/resources/Makefile.am	2006-03-06 17:37:13.000000000 +0100
@@ -2,6 +2,6 @@
 CLEANFILES = $(pkgdata_DATA)
 
 todo.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/todo.zip *.xrc > /dev/null
+	zip todo.zip *.xrc > /dev/null
 
 EXTRA_DIST = add_todo.xrc settings.xrc
--- codeblocks-1.0rc2/src/plugins/contrib/codestat/resources/Makefile.am.orig	2006-03-06 17:36:45.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/contrib/codestat/resources/Makefile.am	2006-03-06 17:36:51.000000000 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 codestat.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/codestat.zip *.xrc > /dev/null
+	zip codestat.zip *.xrc > /dev/null
--- codeblocks-1.0rc2/src/plugins/contrib/profiler/resources/Makefile.am.orig	2006-03-06 17:36:28.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/contrib/profiler/resources/Makefile.am	2006-03-06 17:36:34.000000000 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 profiler.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/profiler.zip *.xrc > /dev/null
+	zip profiler.zip *.xrc > /dev/null
--- codeblocks-1.0rc2/src/plugins/classwizard/resources/Makefile.am.orig	2006-03-06 17:36:14.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/classwizard/resources/Makefile.am	2006-03-06 17:36:21.000000000 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 class_wizard.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/class_wizard.zip *.xrc > /dev/null
+	zip class_wizard.zip *.xrc > /dev/null
--- codeblocks-1.0rc2/src/plugins/defaultmimehandler/resources/Makefile.am.orig	2006-03-06 17:36:00.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/defaultmimehandler/resources/Makefile.am	2006-03-06 17:36:07.000000000 +0100
@@ -2,6 +2,6 @@
 CLEANFILES = $(pkgdata_DATA)
 
 defaultmimehandler.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/defaultmimehandler.zip *.xrc > /dev/null
+	zip defaultmimehandler.zip *.xrc > /dev/null
 
 EXTRA_DIST = edit_files_handling.xrc
--- codeblocks-1.0rc2/src/plugins/compilergcc/resources/Makefile.am.orig	2006-03-06 17:35:47.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/compilergcc/resources/Makefile.am	2006-03-06 17:35:53.000000000 +0100
@@ -2,7 +2,7 @@
 CLEANFILES = $(pkgdata_DATA)
 
 compiler_gcc.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/compiler_gcc.zip *.xrc images/*.png images/16x16/*.png > /dev/null
+	zip compiler_gcc.zip *.xrc images/*.png images/16x16/*.png > /dev/null
 
 EXTRA_DIST = advanced_compiler_options.xrc \
 	compiler_menu.xrc \
--- codeblocks-1.0rc2/src/plugins/codecompletion/resources/Makefile.am.orig	2006-03-06 17:35:30.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/codecompletion/resources/Makefile.am	2006-03-06 17:35:38.000000000 +0100
@@ -2,7 +2,7 @@
 CLEANFILES = $(pkgdata_DATA)
 
 code_completion.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/code_completion.zip *.xrc > /dev/null
+	zip code_completion.zip *.xrc > /dev/null
 
 install-exec-local:
 	mkdir -p $(DESTDIR)$(pkgdatadir)/images/codecompletion
--- codeblocks-1.0rc2/src/plugins/astyle/resources/Makefile.am.orig	2006-03-06 17:35:14.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/astyle/resources/Makefile.am	2006-03-06 17:35:22.000000000 +0100
@@ -4,4 +4,4 @@
 CLEANFILES = $(pkgdata_DATA)
 
 astyle.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/astyle.zip *.xrc > /dev/null
+	zip astyle.zip *.xrc > /dev/null
--- codeblocks-1.0rc2/src/plugins/pluginwizard/resources/Makefile.am.orig	2006-03-06 17:34:59.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/pluginwizard/resources/Makefile.am	2006-03-06 17:35:06.000000000 +0100
@@ -2,6 +2,6 @@
 CLEANFILES = $(pkgdata_DATA)
 
 plugin_wizard.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/plugin_wizard.zip *.xrc > /dev/null
+	zip plugin_wizard.zip *.xrc > /dev/null
 
 EXTRA_DIST = new_plugin.xrc plugin_info.xrc
--- codeblocks-1.0rc2/src/plugins/debuggergdb/resources/Makefile.am.orig	2006-03-06 17:33:17.000000000 +0100
+++ codeblocks-1.0rc2/src/plugins/debuggergdb/resources/Makefile.am	2006-03-06 17:34:31.000000000 +0100
@@ -2,7 +2,7 @@
 CLEANFILES = $(pkgdata_DATA)
 
 debugger_gdb.zip:
-	PWD=`pwd` cd $(srcdir) && zip $(PWD)/debugger_gdb.zip *.xrc images/*.png images/16x16/*.png > /dev/null
+	zip debugger_gdb.zip *.xrc images/*.png images/16x16/*.png > /dev/null
 
 EXTRA_DIST = backtrace.xrc \
 	debugger_menu.xrc \
